packages/mobile/e2e/src/usecases/HandleDeepLinkSend.js

```diff
@@ -17,11 +51,20 @@ export default HandleDeepLinkSend = () => {
     await expect(element(by.id('Review'))).toBeVisible()
 
     // Enter amount and tap review
-    await inputNumberKeypad('1.5')
+    await inputNumberKeypad('0.1')
     await element(by.id('Review')).tap()
 
     // Correct name displayed
-    await expect(element(by.text('Crypto4BlackLives'))).toBeVisible()
+    await expect(element(by.text('TestFaucet'))).toBeVisible()
+
+    // Tap Pay
+    await element(by.text('Send')).tap()
```

Comment:
same as above, should we use id here?

Reply:
fixed in https://github.com/valora-inc/wallet/pull/1913/commits/aa3e205916c5ffe73f85246d4e5b60aa2fe37f5b. I opted for the text selector for TestFaucet since that text is based on the deeplinkrul which is controlled by the test.